common/src/main/scala/uk/ac/wellcome/models/License.scala

```diff
@@ -18,16 +16,20 @@ sealed trait License {
   @JsonProperty("type") val ontologyType: String = "License"
 }
 
-class LicenseDeserialiser extends JsonDeserializer[License] with Logging {
+object License extends Logging {
+  implicit val licenseEncoder = Encoder.instance[License](license => Json.obj(
```

Comment:
Had to do this again cause shapeless doesn't give visibility on things other than case class parameters. Licenses are case objects so they can't have parameters so, meh 😐